XenAdmin/MainWindow.cs

```diff
@@ -1447,7 +1446,7 @@ private void UpdateToolbarsCore()
             NewTabCount = 0;
             ShowTab(TabPageHome, !SearchMode && show_home);
             ShowTab(TabPageGeneral, !multi && !SearchMode && (isVMSelected || (isHostSelected && (isHostLive || !is_connected)) || isPoolSelected || isSRSelected || isVdiSelected || isDockerContainerSelected));
-            ShowTab(dmc_upsell ? TabPageBallooningUpsell : TabPageBallooning, !multi && !SearchMode && (isVMSelected || (isHostSelected && isHostLive) || isPoolSelected));
+            ShowTab(TabPageBallooning, !multi && !SearchMode && (isVMSelected || (isHostSelected && isHostLive) || isPoolSelected));

```

Comment:
Can BallooningUpsellPage class/instance and TabPageBallooningUpsell be removed completely from the MainWindow since they're not used any more?

Reply:
Yes, I'll do it

Reply:
@kc284 pleasee see the new commit